WMF Framework/LocationManagerProtocol.swift

```diff
@@ -0,0 +1,60 @@
+import CoreLocation
+
+public enum LocationManagerAccuracy {
+    /// Location manager with filter `1` accuracy `best`
+    case fine
+    /// Location manager with filter `1000` accuracy `kilometer`
+    case coarse
+}
+
+@objc public enum LocationManagerState: Int {
+    case authorized, notDeterminded, denied, restricted, unknown
+}
+
+@objc public protocol LocationManagerProtocol {
```

Comment:
Add LocationManagerProtocol
This protocol mimics the WMFLocationManager header file. It provides an Objective-C compatible API to the Swift version of LocationManager coming in the next commit.